src/App.tsx

```diff
@@ -1,61 +1,27 @@
 import './App.scss';
+import { useState, FC } from 'react';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList/TodoList';
+import { AddUser } from './components/AddUser/AddUser';
+import { Todo } from './types/types';
+import { getUserById } from './utils/getUserByID';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+export const App: FC = () => {
+  const initialTodos: Todo[] = todosFromServer.map(todo => ({
+    ...todo,
+    user: getUserById(todo.userId),
+  }));
+
+  const [todos, setTodos] = useState<Todo[]>(initialTodos);
 
-export const App = () => {
   return (
     <div className="App">
       <h1>Add todo form</h1>
-
-      <form action="/api/todos" method="POST">
-        <div className="field">
-          <input type="text" data-cy="titleInput" />
-          <span className="error">Please enter a title</span>
-        </div>
-
-        <div className="field">
-          <select data-cy="userSelect">
-            <option value="0" disabled>
-              Choose a user
-            </option>
-          </select>
-
-          <span className="error">Please choose a user</span>
-        </div>
-
-        <button type="submit" data-cy="submitButton">
-          Add
-        </button>
-      </form>
-
-      <section className="TodoList">
-        <article data-id="1" className="TodoInfo TodoInfo--completed">
-          <h2 className="TodoInfo__title">delectus aut autem</h2>
-
-          <a className="UserInfo" href="mailto:Sincere@april.biz">
-            Leanne Graham
-          </a>
-        </article>
-
-        <article data-id="15" className="TodoInfo TodoInfo--completed">
-          <h2 className="TodoInfo__title">delectus aut autem</h2>
-
-          <a className="UserInfo" href="mailto:Sincere@april.biz">
-            Leanne Graham
-          </a>
-        </article>
-
-        <article data-id="2" className="TodoInfo">
-          <h2 className="TodoInfo__title">
-            quis ut nam facilis et officia qui
-          </h2>
-
-          <a className="UserInfo" href="mailto:Julianne.OConner@kory.org">
-            Patricia Lebsack
-          </a>
-        </article>
-      </section>
+      <AddUser
+        onAdd={newTodo => setTodos(currentTodos => [...currentTodos, newTodo])}
```

Comment:
[CODE STYLE] - Don't repeat yourself. If you see that you copy/paste some block code, maybe, it's time to create a separate variable/function. In your case, you're creating a new array with `[...currentTodos, newTodo]` every time a new todo is added. It would be better to create a separate function for this logic.